Assets/MixedRealityToolkit.SDK/Features/UX/Scripts/Slate/FollowMeToggle.cs

```diff
@@ -12,25 +12,25 @@ public class FollowMeToggle : MonoBehaviour
         [Tooltip("An opional object for visualizing the carry mode state")]
         private GameObject visualizationObject = null;
 
-        private Orbital orbital = null;
+        private RadialView radialView = null;
 
         private void Start()
         {
             // Get Orbital Solver component
-            orbital = GetComponent<Orbital>();
+            radialView = GetComponent<RadialView>();
```

Comment:
Since it appears this class requires a RadialView present to do anything. Maybe we should require one? https://docs.unity3d.com/ScriptReference/RequireComponent.html

Reply:
I think it does looook to be optional (i.e. you could in theory have a slate that doesn't follow you, like in the hand interaaction example scenes case)

The only thing that might be helpful is, if the ToggleFollowMeBehavior is engaged, to have some sort of warning or logging if radialView is null (meaning it's not been set up correctly for follow me behavior.

Alternatively, we could try to loosen the dependency directly on RadialView (i.e. maybe ANY time of "following" solver would work here, and we could enable/disable a general solver object instead of this specific one). I'm trying to imagine a consumer wanting to have their own follow me behavior and choosing a non-radial solver (or doing their own)

Reply:
I'm not so sure it is optional, since they should remove this component if a solver isn't present on this object. But, I agree the logic should switch to the base solver class. In which case we can't use require component. haha

Reply:
@Cameron-Micka Made it Required component. Thanks!